packages/db/src/network/index.ts

```diff
@@ -14,7 +14,7 @@
 import { logger } from "@truffle/db/logger";
 const debug = logger("db:network");
 
-import type { Provider } from "web3/providers";
+import type { Web3BaseProvider as Provider } from "web3-types";
```

Comment:
Same as [previous comment](https://github.com/trufflesuite/truffle/pull/6013/files#r1184542764)

Reply:
It's just import renamed to avoid a lot of code changes in the file.